prover/server/prover/combined_circuit.go

```diff
@@ -9,8 +9,8 @@ import (
 
 type CombinedCircuit struct {
 	PublicInputHash frontend.Variable `gnark:",public"`
-	Inclusion       InclusionProof    `gnark:",input"`
-	NonInclusion    NonInclusionProof `gnark:",input"`
+	Inclusion       InclusionProof    `gnark:"input"`
```

Comment:
```suggestion
	Inclusion       InclusionProof    `gnark:",input"`
```

It's more idiomatic way to add tags